src/managed/dir.proj

```diff
@@ -19,6 +19,10 @@
       <BuildArgs>$(BuildArgs) /p:BaseOutputPath=$(IntermediateOutputForPackaging)</BuildArgs>
     </PropertyGroup>
 
-    <Exec Command="$(DotnetToolCommand) build $(BuildArgs) %(PackageProjects.Identity)" />
+    <PropertyGroup>
+      <PackageManagedComponents>false</PackageManagedComponents>
+    </PropertyGroup>
+
+    <Exec Command="$(DotnetToolCommand) build $(BuildArgs) %(PackageProjects.Identity)" Condition="'$(PackageManagedComponents)' == 'true'"/>
```

Comment:
@weshaggard I believe this should stop them from being generated, running a build now to confirm